Update README.md (#1)
podman-desktop/.github: README.md

- 👾 [Fix Bugs/Develop Features](https://github.com/podman-desktop/podman-desktop/blob/main/CONTRIBUTING.md#working-on-issues): Explore the issue tracker, fork the repository, make changes, and submit pull requests.
- 🆕 [Propose Features](https://github.com/podman-desktop/podman-desktop/issues/new?template=enhancement.yml): Suggest and contribute new functionality, UI enhancements, or integrations.
- 🛠️ [Build Extensions](https://podman-desktop.io/docs/extensions/developing): Add missing integrations by developing plug-ins using the provided guide.
- 📝 [Improve Documentation](https://github.com/podman-desktop/podman-desktop/tree/main/website/docs): Enhance existing docs, create guides, or write tutorials.
- 💻 [Contribute to the Website](https://github.com/podman-desktop/podman-desktop/blob/main/WEBSITE_CONTRIBUTING.md): Help improve the website’s content, design, or translations.
- 🦭 [Become an Adopter](https://github.com/podman-desktop/podman-desktop/blob/main/ADOPTERS.md): Showcase your use of Podman Desktop by joining the adopters list.

> Check the [contribution guidelines](https://github.com/podman-desktop/podman-desktop/blob/main/CONTRIBUTING.md) and code architecture for guidance, and join the Kubernetes Slack (#podman-desktop) to collaborate with contributors.

Author: @Firewall @slemeur , Do we have any resource that covers the code architecture of PD for reference here?

Reviewer: The contribution guidelines you link here I would say mention code architecture already: https://github.com/podman-desktop/podman-desktop/blob/main/CONTRIBUTING.md#code-architecture

and also https://podman-desktop.io/docs/extensions/developing for developing an extension

### Additional resources
 

> - [Project governance](https://docs.google.com/document/d/1onDTJxGjVZUuqX1Y7MHgvqf9t9_L7oAHcXPIserX100/edit?tab=t.0#heading=h.n27i858hqauw)

Reviewer: I would remove the google docs link for now. This is not useful for anyone outside of redhat. We can update the link in the future.

Author: Removed the link.

### Your contribution matters

🚨 [Report Issues](https://github.com/podman-desktop/podman-desktop/blob/main/CONTRIBUTING.md#reporting-issues): Share bugs or improvement suggestions on GitHub, including details and reproduction steps.
👾 [Fix Bugs/Develop Features](https://github.com/podman-desktop/podman-desktop/blob/main/CONTRIBUTING.md#working-on-issues): Explore the issue tracker, fork the repository, make changes, and submit pull requests.
🆕 [Propose Features](https://github.com/podman-desktop/podman-desktop/issues/new?template=enhancement.yml): Suggest and contribute new functionality, UI enhancements, or integrations.
🛠️ [Build Plug-ins](https://podman-desktop.io/docs/extensions/developing): Add missing integrations by developing plug-ins using the provided guide.
📝 [Improve Documentation](https://github.com/podman-desktop/podman-desktop/tree/main/website/docs): Enhance existing docs, create guides, or write tutorials.
💻 [Contribute to the Website](https://github.com/podman-desktop/podman-desktop/blob/main/WEBSITE_CONTRIBUTING.md): Help improve the website’s content, design, or translations.

> 🦭 [Become an Adopter](https://github.com/podman-desktop/podman-desktop/blob/main/ADOPTERS.md): Showcase your use of Podman Desktop by joining the adopters list.

Reviewer 2: FYI these lines are not rendered with new lines

# Podman Desktop - A podified solution to developer needs

### Links
   
[Installation](https://podman-desktop.io/docs/installation) | [Documentation](https://podman-desktop.io/docs/intro) | [How to Contribute](https://github.com/podman-desktop/podman-desktop/blob/main/CONTRIBUTING.md) | [Chat on Discord](https://discord.com/invite/x5GzFF6QH4) | [Current Sprint](https://github.com/orgs/containers/projects/4/views/8)

![Image](https://github.com/user-attachments/assets/2f9d933f-2a08-47f7-b4ca-9f7cc5acb873)

### About
Podman Desktop is a developer-friendly open source tool that offers robust and scalable features to work with images, containers, pods, and Kubernetes. Get an intuitive interface to effortlessly bridge the gap between local development and Kubernetes clusters. Build, manage, and deploy your application with ease from a single tool.

> Our mission is to enhance and simplify the use of Podman-native capabilities among developers as they transition from their native development environment to a podified or Kubernetes-environment. The goal is to help them succeed with a unified solution that satisfies most of their development needs.

Reviewer: ```suggestion
Our mission is to enhance and simplify the use of Podman-native capabilities among developers as they transition from their development environment to a podified or Kubernetes-environment. The goal is to help them succeed with a unified solution that satisfies most of their development needs.
```

As Podman is often recommended as the ideal middle ground, providing robust yet lightweight container capabilities, while maintaining alignment with Kubernetes, Podman Desktop provides a gradual experience for developers to build their expertise. 

### Key features

> - [Build, pull, and push images](/docs/containers/images)

Reviewer: None of these links work for me

- [Build, run, and manage containers](/docs/containers)
- [Compatibility with Docker and Compose](/docs/migrating-from-docker)
- [Build your Kubernetes environment](/docs/kubernetes) 
- [Enhance Podman Desktop by adding new features through its extensions](/docs/extensions)
- [Work with LLMs using the Podman AI Lab extension](/docs/ai-lab)

Learn more in [Podman Desktop Documentation](/docs/intro).

### Quick Install
Podman Desktop is available on Windows, macOS, and Linux. Find your favorite way to download and install it [here](/docs/installation). 

### Getting Started

> Need to quickly learn the tool and get an overview of its capabilities, view the links:

Reviewer: ```suggestion
Need to quickly learn the tool and get an overview of its capabilities, check out these links:
```

- [Discover Podman Desktop](/docs/discover-podman-desktop) - Explore key features of Podman Desktop
- [Tutorials](/tutorial) - Step-by-step guide to getting started with Podman Desktop
- [Blogs](/blog) - Know about new releases and additional use cases of Podman Desktop

### Your contribution matters

- 🚨 [Report Issues](https://github.com/podman-desktop/podman-desktop/blob/main/CONTRIBUTING.md#reporting-issues): Share bugs or improvement suggestions on GitHub, including details and reproduction steps.
- 👾 [Fix Bugs/Develop Features](https://github.com/podman-desktop/podman-desktop/blob/main/CONTRIBUTING.md#working-on-issues): Explore the issue tracker, fork the repository, make changes, and submit pull requests.
- 🆕 [Propose Features](https://github.com/podman-desktop/podman-desktop/issues/new?template=enhancement.yml): Suggest and contribute new functionality, UI enhancements, or integrations.

> - 🛠️ [Build Plug-ins](https://podman-desktop.io/docs/extensions/developing): Add missing integrations by developing plug-ins using the provided guide.

Reviewer: ```suggestion
- 🛠️ [Build Extensions](https://podman-desktop.io/docs/extensions/developing): Add missing integrations by developing plug-ins using the provided guide.
```

- 📝 [Improve Documentation](https://github.com/podman-desktop/podman-desktop/tree/main/website/docs): Enhance existing docs, create guides, or write tutorials.
- 💻 [Contribute to the Website](https://github.com/podman-desktop/podman-desktop/blob/main/WEBSITE_CONTRIBUTING.md): Help improve the website’s content, design, or translations.
- 🦭 [Become an Adopter](https://github.com/podman-desktop/podman-desktop/blob/main/ADOPTERS.md): Showcase your use of Podman Desktop by joining the adopters list.

Check the [contribution guidelines](https://github.com/podman-desktop/podman-desktop/blob/main/CONTRIBUTING.md) and code architecture for guidance, and join the Kubernetes Slack (#podman-desktop) to collaborate with contributors.

### Additional resources
 

> Project governance

Reviewer: missing link

> Other community resources

Reviewer: missing link

Author: @slemeur, Do we have any other community resources that we would like to highlight in the additional resources section? Also, i have added this [governance doc](https://docs.google.com/document/d/1onDTJxGjVZUuqX1Y7MHgvqf9t9_L7oAHcXPIserX100/edit?tab=t.0#heading=h.n27i858hqauw) link, but not sure whether it is the final copy or WIP.

Reviewer: The governance document will move to a markdown file into the repository eventually, the google doc was to get some feedback internally first.
